cardano-crypto-praos/src/Cardano/Crypto/VRF/Praos.hs

```diff
@@ -0,0 +1,464 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ForeignFunctionInterface #-}
+{-# LANGUAGE LambdaCase #-}
+
+-- | Mock implementations of verifiable random functions.
+module Cardano.Crypto.VRF.Praos
+  (
+  -- * VRFAlgorithm API
+    PraosVRF
+
+  -- * Low-level size specifiers
+  --
+  -- Sizes of various value types involved in the VRF calculations. Users of
+  -- this module will not need these, we are only exporting them for unit
+  -- testing purposes.
+  , crypto_vrf_proofbytes
+  , crypto_vrf_publickeybytes
+  , crypto_vrf_secretkeybytes
+  , crypto_vrf_seedbytes
+  , crypto_vrf_outputbytes
+
+  -- * Value types
+  --
+  -- These types are all implemented as transparent references. The actual
+  -- values are kept entirely in C memory, allocated when a value is created,
+  -- and freed when the value's finalizer runs.
+  , Seed
```

Comment:
Do we need to export these? Aren't they basically already wrapped by the various `SignKeyVRF` etc types?

Reply:
Indeed, removing exports.